best-practices.md

```diff
@@ -113,6 +114,27 @@ But the stronger recommendation is to host a STAC Browser on your own domain, an
 design to look and feel like your main web presence. STAC aims to be decentralized, so each STAC-compliant data catalog 
 should have its own location and just be part of the wider web.
 
+### Requester Pays
+
+It is very common that large, freely available datasets are set up with a 'requester pays' configuration. This is an option 
+[on AWS](https://docs.aws.amazon.com/AmazonS3/latest/userguide/RequesterPaysBuckets.html)) and [on 
+Google Cloud](https://cloud.google.com/storage/docs/requester-pays), that enables data providers to make their data freely
+available to everyone, while placing the cost of '[egress](https://www.hostdime.com/blog/data-egress-fees-cloud/)' 
+on to the user. For popular datasets that are large in size the egress costs can be substantial, to the point where much
+less data would be available if the cost of distribution was always on the data provider.
+
+For data providers using STAC with requester pays buckets there are two main recommendations:
+
+1) Put the STAC JSON in a separate bucket that is public for everyone, not requestor pays. This enables the STAC metadata
+to be far more crawlable and searchable, but the cost of the egress of STAC files should be miniscule compared to that of
+the actual data. The STAC community can help you work with cloud providers for potential free hosting if you are doing open
```

Comment:
I'm a little hesitant to put this in here, since AWS wouldn't support the MODIS opendata bucket.

Reply:
I wasn't aware of that. I'm happy to help push on that one with AWS. And I think that between Microsoft and Google it's likely to get some free hosting. I think it may be worth leaving in so we can at least try as a community, but I don't think it's on a specific provider. I think I also meant it to be more on the 'stac only' free hosting.

Reply:
Okay, I'm fine with as long as it's something that can actually happen.

AWS (Flasher) passed on it b/c of some other work NASA/USGS was doing to make data available on AWS, which I don't ever actually remember seeing realized.

Reply:
Ok, I just updated the language a bit - the other key point is we can try with microsoft, google and aws. I think at this moment microsoft is the 'best', and I assume that will shift between the three over time, and perhaps there will be others as well.